infrastructure/azure.tf

```diff
@@ -845,7 +845,7 @@ resource "azurerm_log_analytics_workspace" "azurerm_log_analytics" {
   name                = "${local.azurerm_log_analytics_name}"
   location            = "${azurerm_resource_group.azurerm_resource_group.location}"
   resource_group_name = "${azurerm_resource_group.azurerm_resource_group.name}"
-  sku                 = "Standard"
+  sku                 = "pernode"
```

Comment:
it looks like pernode tier does not support deploy to vnet and (but it's not clear, you have to check someway) 
integration with ADB2C:

https://azure.microsoft.com/it-it/pricing/details/api-management/

Reply:
I'm aligning the tf conf to the actual resources, later we'll see to change them as we need